gradio/layouts.py

```diff
@@ -96,6 +96,21 @@ class Tabs(BlockContext):
     Tabs context.
     """
 
+    def __init__(self, selected=False, **kwargs):
```

Comment:
Doesn't seem pythonic to have selected default to `False`. Can we have it default to `None` instead?

```suggestion
    def __init__(self, selected: Optional[int | str] = None, **kwargs):
```

Reply:
The `selected` parameter needs documentation